content/docs/csidriver/features/powerstore.md

```diff
@@ -425,6 +425,16 @@ kubectl get nodes --show-labels
 
 For any additional information about the topology, see the [Kubernetes Topology documentation](https://kubernetes-csi.github.io/docs/topology.html).
 
+## Volume Limit
+
+The CSI Driver for Dell PowerStore allows users to specify the maximum number of PowerStore volumes that can be used in a node.
+
+The user can set the volume limit for a node by creating a node label `max-powerstore-volumes-per-node` and specifying the volume limit for that node.
+<br/> `kubectl label node <node_name> max-powerstore-volumes-per-node=<volume_limit>`
+
+The user can also set the volume limit for all the nodes in the cluster by specifying the same to `maxPowerstoreVolumesPerNode` attribute in values.yaml.
```

Comment:
@alankar-verma Also mention here how to set the volume limit in case of driver installed via operator.